features/in_onderzoek.feature

```diff
@@ -346,574 +413,709 @@ Functionaliteit: in onderzoek
+      | inOnderzoek.leeftijd                                | true         |
+      | inOnderzoek.datumIngangOnderzoekPersoon.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoekPersoon.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoekPersoon.langFormaat | 7 maart 2022 |
 
       Voorbeelden:
       | reden                        | waarde |
       | geboortedatum van de persoon | 010310 |
       | groep geboorte               | 010300 |
       | categorie persoon            | 010000 |
 
-    @proxy
     Abstract Scenario: indicatieOverleden van de persoon in onderzoek omdat <gegeven in onderzoek> is in onderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon heeft volgende 'overlijden' gegevens
-      | naam  | waarde   |
-      | datum | 20220218 |
-      En de persoon heeft volgende 'overlijden.inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
-      Als personen wordt gezocht met de volgende parameters
+      Gegeven de persoon met burgerservicenummer '000000528' heeft de volgende 'overlijden' gegevens
+      | naam                             | waarde   |
+      | aanduiding in onderzoek (83.10)  | <waarde> |
+      | datum ingang onderzoek (83.20)   | 20220307 |
+	  | datum overlijden (08.10)         | 20220218 |
+      Als gba personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
+      | burgerservicenummer | 000000528                       |
       | fields              | overlijden.indicatieOverleden   |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'overlijden.inOnderzoek' gegevens
-      | naam                             | waarde       |
-      | indicatieOverleden               | true         |
-      | datumIngangOnderzoek.datum       | 2022-03-07   |
-      | datumIngangOnderzoek.type        | Datum        |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
+      Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+      | naam                                         | waarde       |
+      | indicatieOverleden                           | true         |
+      | inOnderzoek.indicatieOverleden               | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
 
       Voorbeelden:
       | reden                | waarde |
       | datum overlijden     | 060810 |
```

Comment:
zeggen we dus hiermee dat als plaats of land overlijden in onderzoek zijn dat indicatieOverleden en inOnderzoek.indicatieOverleden niet de waarde true hebben?

Reply:
Ik denk dat je deze conclusie niet mag trekken alleen omdat die voorbeelden hier ontbreken. Ik zou deze voorbeelden nog wel toe kunnen voegen.
Als plaats van overlijden in onderzoek is (code 060820) of als land van overlijden in onderzoek is (code 060830) dan krijgt 'inOnderzoek.indicatieOverleden' volgens 'in onderzoek.xlsx' niet de waarde true. Ik heb dat overzicht hier gevolgd. 
Het lijkt me dat 'indicatieOverleden' dan wel de waarde 'true' zou moeten hebben.